pkg/users/password-dialogs.js

```diff
@@ -171,33 +145,16 @@ function SetPasswordDialogBody({ state, errors, change }) {
                 <TextInput className="check-passwords" type="password" id="account-set-password-old"
                            value={password_old} onChange={value => change("password_old", value)} />
```

Comment:
Why is "password_old" here as a string? That seems like bug.

Reply:
What do you mean password_oldl is a string? In my browser I see the usual *** filling the password field while I type.

Reply:
oh, nvm, I looked at this wrongly